QQE-1290.md

```diff
@@ -0,0 +1,19 @@
+# QQE-1290 Test RestClientReactive base URL override
+
+JIRA link: https://issues.redhat.com/browse/QQE-1290
+
+Test plan covers reactive rest client's ability to override base URL.
+
+## Scope of the testing
+Verify that quarkus-reactive-rest-client is able to override base URL by annotation-based parameters on runtime.
```

Comment:
`quarkus-reactive-rest-client` does not exist in main or in 3.15. In 3.8 there was `quarkus-rest-client-reactive`, but the name was different to what you are referring to.